grammars/css.cson

```diff
@@ -1913,7 +1913,8 @@
             'match': '''(?x)
               (-?(?!\\d)(?:[\\w-]|[^\\x00-\\x7F]|\\\\(?:[0-9a-fA-F]{1,6}|.))+)
               \\s*
-              (?=[~|^\\]$*=]|/\\*)
```

Comment:
What the HELL, that quantifier shouldn't be there. :| Oops.

@nathansobo Try removing `$*`, that's a zero-length match that'll match anything. Really bad news.